.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: build
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - Language/**
+      - tests/Language/**
+      - .github/workflows/build.yml
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - Language/**
+      - tests/Language/**
+      - .github/workflows/build.yml
+
+jobs:
+
+  main:
+    name: Build [PHP ${{ matrix.php-versions }}]
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@master
```

Comment:
Please use `v2`. I started the whole `master` bit, my fault, but it is a really bad idea and needs to die.